filament/src/details/Camera.cpp

```diff
@@ -286,14 +286,13 @@ CameraInfo::CameraInfo(FCamera const& camera, const math::mat4& worldOriginCamer
     cullingProjection  = mat4f{ camera.getCullingProjectionMatrix() };
     model              = mat4f{ modelMatrix };
     view               = mat4f{ inverse(modelMatrix) };
+    worldOrigin        = worldOriginCamera;
```

Comment:
Why not add a `userViewMatrix` field here instead of having to "undo" the world origin later in `getUserViewMatrix()`?

Reply:
a-ha, good question! In fact this change was partly so that I could do that easily in the future! :-)
The downside would be that this structure (which is already big) would increase.

Reply:
👍 got it!